CHANGELOG.md

```diff
@@ -1,4 +1,106 @@
 # CHANGELOG
+## v0.2.0 | 2022-02-23
+The second minor release focuses on the stability and robustness of existing features via minor refactorings, mainly for
```

Comment:
> via minor refactorings 

I think we can omit `minor` as down below we are referring to a "almost rewritten from scratch with a better design"

Reply:
Can you make a suggestion?

Reply:
```suggestion
The second minor release focuses on the stability and robustness of existing features via refactorings, mainly for
```